app/assets/v2/js/grants/_new.js

```diff
@@ -277,7 +277,26 @@ Vue.mixin({
           _alert(err.message, 'danger');
         }
       });
-    }
+    },
+    onBlur(event) {
+      let vm = this;
+      
+      event.preventDefault();
+
+      const inputField = event.target;
+      const fullTwitterURL = /https:\/\/twitter.com\/\w{1,15}/
+      const twitterUsername = /(\w{1,15})$/
```

Comment:
are twitter handles limited to 15 characters?

Reply:
Yep https://help.twitter.com/en/managing-your-account/twitter-username-rules

Reply:
It also specifies it must be at least 4 characters, it might be worth mentioning to Frank and adding a ticket to handle that (or add it to this one).